corehq/apps/accounting/views.py

```diff
@@ -338,8 +338,27 @@ def cancel_form(self):
         return CancelForm()
 
     @property
-    def page_context(self):
+    def invoice_context(self):
+        subscriber_domain = self.subscription.subscriber.domain
+
+        invoice_report = InvoiceInterface(self.request)
+        invoice_report.filters.update(subscription__subscriber__domain=subscriber_domain)
+
+        invoice_report_url = invoice_report.get_url()
+        invoice_export_url = invoice_report.get_url(render_as='export')
+        # Tied to InvoiceInterface. Not a neat way of constructing URL with GET params
+        get_params_string = "?subscriber={domain_name}".format(domain_name=subscriber_domain)
```

Comment:
You should be able to do this:
```python
import urllib
get_params_string = "?" + urllib.urlencode({'subscriber': subscriber_domain})
```
That'll escape stuff as needed to put in a URL, although that's probably not an issue with the restrictions we put on domain names